test/Integration/SqlTriggerBindingIntegrationTests.cs

```diff
@@ -269,9 +269,15 @@ void OutputHandler(object sender, DataReceivedEventArgs e)
 
             // All trigger integration tests are only using C# functions for testing at the moment.
             this.StartFunctionHost(functionName, Common.SupportedLanguages.CSharp, useTestFolder, OutputHandler);
+
+            // The functions host generally logs the error message within a second after starting up.
+            const int BufferTimeForErrorInSeconds = 5;
```

Comment:
I'd suggest at least 15 seconds for this, I've had host startups take many seconds before and assuming the test works correctly it doesn't really matter what this value is since it shouldn't be hitting it much.